en/scripting/modules/engine.md

```diff
@@ -0,0 +1,85 @@
+# Modules
+
+The engine and editor expose functional interfaces to developers via modules, which exist as **ECMAScript** modules.
+
+> **Note**: As of 3.0, engine functions will not be accessible via global variables prefixed with `cc`!
```

Comment:
As of 3.0, engine
-->
Engine

I dont feel we need to include v3 info as we are in the v3 docs when viewing this.

Reply:
@slackmoehrle Maybe ”engine“